settings.talon

```diff
@@ -1,73 +1,73 @@
 settings():
-    # Adjust the scale of the imgui to my liking
+    # Adjust the scale of the imgui
     imgui.scale = 1.3
 
-    # Enable if you'd like the picker gui to automatically appear when explorer has focus
+    # Automatically show the picker GUI when the file manager has focus
     user.file_manager_auto_show_pickers = false
```

Comment:
If the value is `false`, shouldn't this comment read "Don't automatically..."?

Reply:
Yes, if as before, we want comments to describe the default value. Since it is much clearer when something is on/off with true/false, I've changed them to describe the functionality: in this case, `true` automatically shows the picker whereas `false` does not. 

I find "Don't automatically..." more confusing: does `false` not show the GUI? Or does it not...not show it.

If you agree with the approach, I can make each comment more explicit: "Automatically show the picker GUI when the file manager has focus. Default is `false`."

Reply:
maybe `# If `true`, automatically ...`?